### ISSUE TITLE
Dead link in README

### ISSUE THREAD

**Issue by TomazErjavec** [TEIC/Stylesheets#703]
The README.md states that "It is helpful to have the TEI environment installed locally. Please refer to http://www.tei-c.org/Guidelines/P5/get.xml for hints on how to do that." 
However, http://www.tei-c.org/Guidelines/P5/get.xml  is a dead link.

**Comment by ebeshero** [comment]
After some searching on the old and new websites, I think I've found it. I don't think we've had get.xml in quite some time, but we have instructions on accessing the TEI locally on this page: https://tei-c.org/guidelines/p5/  

~~I'll just go and add that now!~~

**Comment by ebeshero** [comment]
No--I won't be adding that. It makes sense because it's the one place where we mention setting up a local environment for working with the TEI Guidelines, so I think that is why @rettinghaus and I both reached for that link. But we want something that comments on the local setup environment for Stylesheets and Guidelines...I wonder if this is in the Vault, or whether it's time to write a new version.

**Comment by ebeshero** [comment]
Aha! I think it's TCW 32 that we need, yes? https://tei-c.org/documentation/tcw32/

But TCW 32 _also_ has a dead link to the mysterious get.xml, referring to it as "Using the TEI Guidelines": see https://github.com/TEIC/Documentation/blob/b9e7e139957339fc1cc06c5a48565cfaa6e65615/TCW/tcw32.xml#L105

**Comment by rettinghaus** [comment]
The document is still available under <https://old.tei-c.org/guidelines/p5/using-the-tei-github-repository/>. I think this is more how to (simply) set up everything locally.

So is <https://tei-c.org/documentation/tcw32/>.

If you need to know how to clone a repo or something else, there is plenty information on GitHub itself. 

I think the best replacement for now is still the overview: <https://tei-c.org/guidelines/p5/>.

**Comment by ebeshero** [comment]
Concurring with @rettinghaus : I think we need links to TCW32 together with the Guidelines page, but we *also* need to do some rewriting and updating.
